programs/locker/src/instructions/v3/cancel_vesting_escrow.rs

```diff
@@ -0,0 +1,136 @@
+use anchor_spl::memo::Memo;
+use anchor_spl::token_interface::{
+    close_account, CloseAccount, Mint, TokenAccount, TokenInterface,
+};
+use util::{
+    harvest_fees, parse_remaining_accounts, transfer_to_user_v3, AccountsType,
+    ParsedRemainingAccounts, TRANSFER_MEMO_CANCEL_VESTING,
+};
+use crate::util::MemoTransferContext;
+use crate::*;
+
+/// Accounts for [locker::cancel_vesting_escrow_v3].
+#[derive(Accounts)]
+#[event_cpi]
+pub struct CancelVestingEscrowV3<'info> {
+    /// Escrow.
+    #[account(
+        mut,
+        has_one = token_mint,
+        constraint = escrow.load()?.cancelled_at == 0 @ LockerError::AlreadyCancelled
+    )]
+    pub escrow: AccountLoader<'info, VestingEscrowV3>,
+
+    /// Mint.
+    #[account(mut)]
```

Comment:
Do we need `mut` for signer?

Reply:
As i know, signer will pay fee for transaction. So i usually marked it `mut`.

Reply:
i removed `mut` for `signer` and also remove mut for `recipient` in `claim_v3` instruction.